lib/mayaUsd/fileio/registryHelper.cpp

```diff
@@ -337,7 +336,7 @@ void UsdMaya_RegistryHelper::AddUnloader(const std::function<void()>& func, bool
 {
     if (fromPython) {
         g_pythonUnloaders.emplace_back(func);
-        if (boost::python::import("atexit")
+        if (PXR_BOOST_PYTHON_NAMESPACE::import("atexit")
```

Comment:
You'll see a ton of changes like this. Removing any `<boost/python/xxx>` includes replacing with new `<pxr_python.h>` and using define for namespace. This handles pre 24.11 and >= 24.11